README.md

```diff
@@ -11,6 +11,8 @@ We've been using Emacs for years here at [Frontside][frontside], and
 have finally decided to share the configuration for our favorite
 editor with the world. Why did we make our own?
 
+![screenshot](https://raw.githubusercontent.com/rhoit/frontmacs/dump/screenshot.png)
```

Comment:
You might consider using `https://user-images.githubusercontent.com/896684/34786587-6806f550-f5e9-11e7-8c25-151968084076.png` (see my comment on this PR for more info)